adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala

```diff
@@ -229,6 +229,15 @@ case class ReferenceRegion(referenceName: String, start: Long, end: Long, orient
   def length(): Long = {
     end - start
   }
+
+  override def hashCode: Int = {
+    var result = 37
+    result = 41 * result + (if (referenceName != null) referenceName.hashCode else 0)
+    result = 41 * result + (if (start != null) start.hashCode else 0)
```

Comment:
Does this mean that two `ReferenceRegion` objects that differ only in that one has `start=0` and the other has `start=null` would have the same `hashCode`?